### PR TITLE
Origin/main

### PR DESCRIPTION
**Pull request by e4t**
Update Slurm upgrade chapter
Fix minor issue

### CODE REVIEW

**Comment by taroth21** [comment]
@KucharczykL : Could you review this one and merge, please? It is related to the RN submission by @e4t, please take care to keep them in sync.

File: xml/slurm.xml [hunk 1]
```diff
@@ -847,6 +847,14 @@ Description="subgroup" Organization=bavaria</screen>
   upgrade and replace deprecated configuration options before the final restart
   of a service.
  </para>
+ <para>
+  It should be noted that a new major version of Slurm introduces a new version
+  of <literal>libslurm</literal>. Older versions of this library may no longer
```
Comment (KucharczykL) [inline xml/slurm.xml:852]: We only use `may` to express permissions. For expressing possibility we use `might`. See https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-vocabulary

```suggestion
  of <literal>libslurm</literal>. Older versions of this library might no longer
```

File: xml/slurm.xml [hunk 2]
```diff
@@ -847,6 +847,14 @@ Description="subgroup" Organization=bavaria</screen>
...
+  work with an upgraded Slurm. For all SLE software depending on
```
Comment (KucharczykL) [inline xml/slurm.xml:853]:
```suggestion
  work with an upgraded Slurm. For all &slea; software depending on
```

File: xml/slurm.xml [hunk 3]
```diff
@@ -905,6 +913,13 @@ Description="subgroup" Organization=bavaria</screen>
        Make sure that <literal>slurmdbd</literal> is not running anymore:
       </para>
       <screen>&prompt;rcslurmdbd status</screen>
+      <para>
+       It should be noted that <literal>slurmctld</literal> may remain running
```
Comment (KucharczykL) [inline xml/slurm.xml:917]:
```suggestion
       It should be noted that <literal>slurmctld</literal> might remain running
```

File: xml/slurm.xml [hunk 4]
```diff
@@ -905,6 +913,13 @@ Description="subgroup" Organization=bavaria</screen>
...
+       while the database daemon is down. While it is down requests intended
```
Comment (KucharczykL) [inline xml/slurm.xml:918]:
```suggestion
       while the database daemon is down. While it is down, requests intended
```

File: xml/slurm.xml [hunk 5]
```diff
@@ -989,6 +1004,11 @@ success!</screen>
        <literal>SIGTERM</literal> (that is, by pressing
        <keycombo><keycap function="control"/><keycap>C</keycap></keycombo>).
       </para>
+      <para>
+       When using a backup slurmdbd, the conversion needs to be performed
```
Comment (KucharczykL) [inline xml/slurm.xml:1008]:
```suggestion
       When using a backup <literal>slurmdbd</literal>, the conversion needs to be performed
```

File: xml/slurm.xml [hunk 6]
```diff
@@ -1124,17 +1144,18 @@ SlurmdTimeout=3600</screen>
...
       </para>
       <para>
-       Old versions will not understand a newer format, thus it is useful to
-       back up the old state in case an update needs to be rolled back.
-       Otherwise the content will not be understood on downgrade and will be
-       discarded This will in turn lead to the loss of all pending and running
-       jobs.
+       However, if a downgrade should become necessary, state information
+       from newer versions will not be recognized by an older version of
+       slurmctld and thus will be discarded, resulting in a loss of all
```
Comment (KucharczykL) [inline xml/slurm.xml:1156]:
```suggestion
       <literal>slurmctld</literal> and thus will be discarded, resulting in a loss of all
```

**Pull request merged**